ex03.ex

```diff
@@ -59,8 +59,11 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
-
+  def odd_even([]), do: []
+  def odd_even([h | t]), do: [is_even_update(Integer.is_even(h)) | odd_even(t)]
```

Comment:
No deduction, but a design note.

The code that takes an integer and converts it into an atom is split across two functions, `odd_even` and the somewhat strangely named `odd_even_update`. Really, though, that's a single transformation, int -> atom, and so it should be split off into its own function. That function may then choose to use helpers to do the actual atom selection.